packages/expo-contacts/src/Contacts.ts

```diff
@@ -307,11 +307,11 @@ export async function addContactAsync(contact: Contact, containerId: string): Pr
   return await ExpoContacts.addContactAsync(contact, containerId);
 }
 
-export async function updateContactAsync(contact: Contact): Promise<string> {
+export async function updateContactAsync(contact: Contact, fields?: FieldType[]): Promise<string> {
   if (!ExpoContacts.updateContactAsync) {
     throw new UnavailabilityError('Contacts', 'updateContactAsync');
   }
-  return await ExpoContacts.updateContactAsync(contact);
+  return await ExpoContacts.updateContactAsync(contact, fields);
```

Comment:
Probably need to add this argument on android as well.

Reply:
@EvanBacon this method is [iOS only](https://docs.expo.io/versions/latest/sdk/contacts/#updatecontactasync)